### ISSUE TITLE
Renaming/Renumbering files in StoryArcs

### ISSUE THREAD

**Issue by war3zlod3r** [mylar3/mylar3#776]
**Describe the request**
It would be nice if there we're an easy way to renumber comics in storyarcs, as an example I don't collect FCBD issues, "Civil War II" starts with one as the first issue forcing me to go back and rename all the individual issues after incrementing down, 

**How did you come up with the idea?**
Frustratingly renumbering 100 issues

**Environment (please complete the following information):**
 - OS: Ubuntu 21.04
 - Branch / commit: main

**Comment by evilhero** [comment]
The renumbering aspect is a bit of a sore spot I'm afraid.

Because CV doesn't have any reading order in their API, Mylar tries to calculate the order based on dates, which if you have 2 on the same date, it gets messy cause then they could flip depending on how the API response is returned from CV. I've tried in the past to do it with the current system, and it's nowhere near clean enough for me to even want to attempt to do something given that it would be manipulating physical files.

Have you tried to use the ```remove``` option to remove the issue from the given storyarc, and then see if it can renumber based on that ? In typing that out, I'm actually not sure if it would do a rename of the existing files or if it would create duplicates of the files with different reading order numbering - which would be worse.

The updated storyarc system, when it comes into play, would allow for this type of manipulating of the reading order since it wouldn't be solely relying on CV to get the actual reading order (meaning it would be consistent and wayy more accurate). Unfortunately there's no timeline for when it'll be available at this point.

**Comment by war3zlod3r** [comment]
You are correct it doesn’t seem to renumber them simply skips the number removed. Looking forward to seeing the new storyarc system launch!

**Comment by barbequesauce** [comment]
Closing due to staleness, we welcome contributions for new features. If this is an issue and you are still experiencing it, please open a new ticket with a care package attached.
